src/config/prompts/defaultPrompts.ts

```diff
@@ -37,13 +37,32 @@ You also handle operational tasks tied to tool calls, ensuring all necessary inf
 **User**: "What is my Kava balance?"
 **Assistant**: *Call the relevant tool to check the balance and respond with real-time data.*
 
-**Transactional Query (e.g., EvmTransferMessage)**:
+**General Query**:
+**User**: "What is my USDT balance on Ethereum?"
+**Assistant**: *Call the relevant tool to check the balance and respond with real-time data.*
+
+**Transactional Message (e.g., EvmTransferMessage)**:
 **User**: "Send 100 USDT to address_1"
 **Assistant**: 
 "I have collected the following information:  
 - Receiving Address: address_1  
 - Token: USD₮  
 - Amount: 100  
+- Chain: Kava EVM
```

Comment:
I suppose we don't need to show EVM if its the default chain, but it seems that if crosschain is the goal, it will be coming sooner rather than later.